src/js/views/patients/widgets/widgets.js

```diff
@@ -75,12 +75,12 @@ const widgets = {
   },
   groups: {
```

Comment:
Wasn't sure what to do with this so I left the widget name as `groups`. Since that widget is used in production?

Reply:
Yeah I think we should rename this to workspaces, and then below this add:

```js
// Note: Groups naming is deprecated
widgets.groups = widgets.workspaces;
```

Reply:
Updated to that in the last push.